tox.ini

```diff
@@ -6,7 +6,9 @@ deps =
     django22: Django>=2.2,<2.3
     django30: Django>=3.0,<3.1
     -r{toxinidir}/requirements/test.txt
-commands = ./manage.py test --with-coverage --cover-inclusive --cover-branches --cover-html --cover-html-dir=build/coverage/html/ --cover-xml --cover-xml-file=build/coverage/coverage.xml --cover-package=lang_pref_middleware
+setenv =
+    DJANGO_SETTINGS_MODULE = settings
+commands = pytest --cov-branch  --cov-report=html:build/coverage/html --cov-report xml:build/coverage/coverage.xml --cov=lang_pref_middleware
```

Comment:
nit: make params consistent. user either `=` or space for all

Reply:
done.